nik4.py

```diff
@@ -602,8 +604,8 @@ def run(options):
     parser.add_argument('--url', help='URL of a map to center on')
     parser.add_argument('--ozi', type=argparse.FileType('w'), help='Generate ozi map file')
     parser.add_argument('--wld', type=argparse.FileType('w'), help='Generate world file')
-    parser.add_argument('-t', '--tiles', type=int, choices=range(1, 13), default=1,
-                        help='Write N×N tiles, then join using imagemagick')
+    parser.add_argument('-t', '--tiles',
```

Comment:
I guess the code above would break when this argument is not specified.

```suggestion
    parser.add_argument('-t', '--tiles', default='1',
```